examples/indexer/java/AccountInfo.java

```diff
@@ -3,25 +3,33 @@
 package com.algorand.javatest.indexer;
 
 import com.algorand.algosdk.v2.client.common.IndexerClient;
+import com.algorand.algosdk.v2.client.common.Response;
 import com.algorand.algosdk.v2.client.common.Client;
 import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.v2.client.model.AccountResponse;
+
 import org.json.JSONObject;
 
 public class AccountInfo {
     public Client indexerInstance = null;
     // utility function to connect to a node
     private Client connectToNetwork(){
         final String INDEXER_API_ADDR = "localhost";
-        final int INDEXER_API_PORT = 8980;       
+        final int INDEXER_API_PORT = 8981;       
```

Comment:
Please revert to 8980 in all instances on this page. This is our standard port assignment for indexer.

Reply:
yes.. change to 8980